File: code/48115.pas
program Bai115;
var xA, xB, yA, yB: integer;
begin
  readln(xA, yA);
  readln(xB, yB);
  if xA = xB then   writeln('AB // OY')
    else if yA = yB then   writeln('AB // OX')
    else  writeln('KHONG');
end.

File: code/48115.cpp
#include <iostream>
using namespace std;

int main() 
{
    int xa,xb,ya,yb;
    cin>>xa>>ya;
    cin>>xb>>yb;
    if (xa==xb)
        cout<<"Oy";
    else if (ya==yb)
        cout<<"Ox";
    else cout<<"KHONG";
    return 0;
}